src/azul/__init__.py

```diff
@@ -649,8 +649,8 @@ def plugin_name(self, catalog_name: CatalogName, plugin_type: str) -> str:
     def subscribe_to_dss(self):
         return self._boolean(os.environ['AZUL_SUBSCRIBE_TO_DSS'])
 
-    service_cache_health_lambda_basename = 'servicecachehealth'
-    indexer_cache_health_lambda_basename = 'indexercachehealth'
+    def lambda_basename(self, app_route):
+        return app_route.func.__name__
```

Comment:
I don't think this works if `@route(…, name='foo')` is used to override the default.